src/optimize/base_optimizer.js

```diff
@@ -64,8 +68,38 @@ class BaseOptimizer {
   }
 
   getConfig() {
-    const mapQ = this.sourceMaps ? '?sourceMap' : '';
-    const mapQPre = mapQ ? mapQ + '&' : '?';
+    const loaderWithSourceMaps = (loader) =>
+      setLoaderQueryParam(loader, 'sourceMap', !!this.sourceMaps);
+
+    const makeStyleLoader = preprocessor => {
+      let loaders = [
+        loaderWithSourceMaps('css')
+      ];
+
+      if (preprocessor) {
+        loaders = [
+          ...loaders,
+          {
+            name: 'postcss',
+            query: {
+              config: require.resolve('./postcss.config')
+            }
+          },
+          loaderWithSourceMaps(preprocessor)
+        ];
+      }
+
+      return ExtractTextPlugin.extract(makeLoaderString(loaders));
+    };
+
+    const makeBabelLoader = query => {
+      return makeLoaderString([
+        {
+          name: 'babel',
+          query: defaults({}, query || {}, babelOptions.webpack)
```

Comment:
Yeah, they were in use before #6177 and turning them on makes the babel config on the front-end and backend consistent.